app/liquid_tags/user_tag.rb

```diff
@@ -13,7 +13,7 @@ def render(_context)
     ActionController::Base.new.render_to_string(
       partial: PARTIAL,
       locals: {
-        user: @user,
+        user: @user.decorate,
```

Comment:
Added `decorate` call here to remove it from the model where it was used more than once.